microsetta_public_api/server.py

```diff
@@ -29,5 +29,12 @@ def build_app(resources_config_json=None):
 if __name__ == "__main__":
     import sys
     config_fp = sys.argv[1] if len(sys.argv) > 1 else None
-    app = build_app(resources_config_json=config_fp)
-    app.run(port=8083, debug=True)
+    if config_fp:
+        app = build_app(resources_config_json=config_fp)
+        app.run(port=8083, debug=True)
+    else:
+        # import TestDatabase here to avoid circular import
+        from microsetta_public_api.utils.testing import TestDatabase
+        with TestDatabase():
+            app = build_app()
+            app.run(port=8083, debug=True)
```

Comment:
```suggestion
    PORT = 8084
    if config_fp:
        app = build_app(resources_config_json=config_fp)
        app.run(port=8084, debug=True)
    else:
        # import TestDatabase here to avoid circular import
        from microsetta_public_api.utils.testing import TestDatabase
        with TestDatabase():
            app = build_app()
            app.run(port=8084, debug=True)
```